### ISSUE TITLE
the new update with high/low is working fine, but...

### ISSUE THREAD

**Issue by DJ-Dingo** [pibooth/pibooth-extra-lights#7]
Hey, @anxuae i just testet the new update with high/low, and it is working fine.
(it was not in the pip install yet).

I do have a little trouble with adding the same pin number to both "preview_led_pin and flash_led_pin", at the same time. I use this from time to time, when my pibooth is setup in a dark room. That way the users can better see themself thrue the preview also. I did found out that the preview also work under the flash stage, so using preview as the flash is the workaround for this most of the time. 

Only problem is, if i simetimes has 2 defferent lights, 1 for preview (a smal sign), and 1 for flash, and i also want the sign to be on at startup stage.

**Comment by anxuae** [comment]
Hi @DJ-Dingo , you can use the startup_led for that, no? Or I missed something in your needs?

**Comment by DJ-Dingo** [comment]
if you want to use the same pin allready used then no. gpiozero will say pin allready in use. But i can use my own version, that fit my needs. it was more a general thing.
And it´s also a good thing to keep this plugin simple.

Btw, you should put this plugin on the pibooth github frontpage along with the others "Pibooth organisation's plugin"

**Comment by DJ-Dingo** [comment]
Hey @anxuae it seems to wotk fine on my main pibooth, but not on my test pibooth.  so I will double check, but think it must be my test pibooth setup where something make it fails when i use the same pin twice.

**Comment by anxuae** [comment]
Thanks for feedback. I've added the plugin to the list.

**Comment by DJ-Dingo** [comment]
@anxuae  After doing some testing, it seems if you have multiple plugins that use gpiozero, this error can occur. For example, on my pibooth test machine, in "manage plugins", I had switched off various plugins that use the same pins, as I knew that this would cause problems.

Pibooth, crashes when turning On the plugins that use the same pin (which is understandable). But it depends a little on the order the plugin is loaded and when they are loaded. If they are all switched On from the start, gpiozero naturally says that pin is already in use. If you switch them On and Off in "manage plugins", everything from crashes to gpiozero saying "pins are allready in use" can happen.

But for some reason it is sometimes enough that they are in the list of plugins that is turned off in "manage plugins, (dont no if they are loadet shortly)". And also sometimes when you just have another plugin using gpiozero, (even that it dont use the same pins) This one is strange because it shouldn't be possible to happen.

So basically there is nothing wrong with this plugin or pibooth. It is a combination of other things. I am closing this,, hope you can use the info for later use.
